src/sfctl/commands.py

```diff
@@ -199,7 +200,11 @@ def load_command_table(self, args): #pylint: disable=too-many-statements
 
             with super_group.group('chaos') as group:
                 group.command('stop', 'stop_chaos')
-                group.command('report', 'get_chaos_report')
+                group.command('events', 'get_chaos_events')
+                group.command('get', 'get_chaos')
+
+            with super_group.group('chaos-schedule') as group:
```

Comment:
I think `sfctl chaos schedule` is better than `sfctl chaos-schedule`. Logically I think schedule should be an additional subgroup under `chaos` rather than a top level group under `sfctl`.